docs/reference/misc/sounds.mdx

```diff
@@ -0,0 +1,160 @@
+---
+id: sounds
+title: Sound Presets
+---
+
+Sounds are used to draw a player's attention to certain events on a map. PGM comes with several preset sounds that can be quickly used for scenarios.
+It can be used in combination with other actions, as defined in [Action &amp; Triggers](/docs/modules/mechanics/actions-triggers#sound-attributes).
```

Comment:
```suggestion
It can be used in combination with other actions, as defined in [Actions &amp; Triggers](/docs/modules/mechanics/actions-triggers#sound-attributes).
```